.devcontainer/base/devcontainer.json

```diff
@@ -1,8 +1,8 @@
 {
-  "name": "Autoware",
+  "name": "autoware:universe",
   "build": {
     "dockerfile": "../Dockerfile",
-    "args": { "BASE_IMAGE": "ghcr.io/autowarefoundation/autoware:latest-devel" }
+    "args": { "BASE_IMAGE": "ghcr.io/autowarefoundation/autoware:universe-devel" }
```

Comment:
Do you test it? The tag doesn't exist yet. 
https://github.com/autowarefoundation/autoware/issues/5175

Reply:
This will remove the `latest` https://github.com/autowarefoundation/autoware/pull/5201